src/jdk.jdwp.agent/share/native/libjdwp/util.c

```diff
@@ -1901,9 +1908,44 @@ map2jvmtiError(jdwpError error)
     return AGENT_ERROR_INTERNAL;
 }
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+static jvmtiExtensionEventInfo *
+find_ext_event(jvmtiEnv *jvmti, const char *ename)
+{
+    jint extCount = 0;
+    jvmtiExtensionEventInfo *extList = NULL;
+
+    jvmtiError err = JVMTI_FUNC_PTR(jvmti, GetExtensionEvents)
+                            (jvmti, &extCount, &extList);
+    if (JVMTI_ERROR_NONE == err) {
+        for (int i = 0; i < extCount; i++) {
+            if (NULL != strstr(extList[i].id, ename)) {
+                return &extList[i];
+            }
+        }
+    } else {
+        ERROR_MESSAGE(("Error in JVMTI GetExtensionEvents: %s(%d)\n", jvmtiErrorText(err), err));
+    }
+    return NULL;
+}
+#endif /* defined(J9VM_OPT_CRIU_SUPPORT) */
+
 static jvmtiEvent index2jvmti[EI_max-EI_min+1];
 static jdwpEvent  index2jdwp [EI_max-EI_min+1];
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
```

Comment:
This will be helpful with the update to `find_ext_event()`
```c
#define OPENJ9_EVENT_VM_RESTORE "openj9.event.VMRestore"
```

Reply:
Use OpenJ9 define.